independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/BeanInfo.java

```diff
@@ -492,6 +492,22 @@ public String getTargetPackageName() {
         return packageName;
     }
 
+    public String getClientProxyPackageName() {
+        if (isProducerField() || isProducerMethod()) {
+            AnnotationTarget target = getTarget().get();
+            DotName typeName = target.kind() == Kind.FIELD ? target.asField().type().name()
+                    : target.asMethod().returnType().name();
+            String packageName = DotNames.packageName(typeName);
+            if (packageName.startsWith("java.")) {
```

Comment:
I recall in Weld we were replacing a regex of `java.*` meaning we'd replace even `javax.something` etc. And latest version also does that for `jakarta` package but TBF I don't remember why we chose/needed to do that...

Reply:
> TBF I don't remember why we chose/needed to do that...

Exactly. That's the reason why we only keep `java.` for now ;-).